dev-games/libsmacker/libsmacker-1.1.1.ebuild

```diff
@@ -0,0 +1,20 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools eutils
```

Comment:
What's using eutils here? Can the inherit be dropped?

Reply:
Will drop it. Had euitls first.. but needed autoreconf --install so added autotools

Reply:
removed